addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -184,19 +184,21 @@
                             </DatiRiepilogo>
                         </t>
                     </DatiBeniServizi>
-                    <DatiPagamento>
+                    <DatiPagamento t-if="record.type != 'out_refund'">
```

Comment:
should not we do and record.invoice_partner_bank_id?  @dbkosky

Reply:
and remove the constraint?

Reply:
Actually, right now, we get an error if we remove the bank account on the invoice 
![Screenshot from 2022-04-14 22-19-12](https://user-images.githubusercontent.com/3832699/163469308-aaa3ea6c-fb65-4ec5-8c74-ea604be38bab.png)

Reply:
but it should have been the case before as well.